README.md

```diff
@@ -570,24 +570,21 @@ Currently we recommend using the above hook to keep track of how many user's rea
 Below is the list of potential events currently being tracked by the hook:
 
 ```
-WELCOME - User reached the "Welcome" screen 
-DOCUMENT_CAPTURE - User reached the "document capture" screen (for one-sided document) 
-DOCUMENT_CAPTURE_FRONT - User reached the "document capture" screen for the front side (for two-sided document) 
-DOCUMENT_CAPTURE_BACK - User reached the "document capture" screen for the back side (for two-sided document) 
-DOCUMENT_CAPTURE_CONFIRMATION - User reached the "document confirmation" screen (for one-sided document) 
-DOCUMENT_CAPTURE_CONFIRMATION_FRONT - User reached the "document confirmation" screen for the front side (for two-sided document) 
-DOCUMENT_CAPTURE_CONFIRMATION_BACK - User reached the "document confirmation" screen for the back side (for two-sided document) 
-DOCUMENT_UPLOAD - User's document is uploading 
-FACIAL_INTRO - User reached the "selfie intro" screen 
-FACIAL_CAPTURE - User reached the "selfie capture" screen 
-FACIAL_CAPTURE_CONFIRMATION - User reached the "selfie confirmation" screen 
-FACIAL_UPLOAD - User's selfie is uploading 
-VIDEO_FACIAL_INTRO - User reached the "liveness intro" screen 
-VIDEO_FACIAL_CAPTURE - User reached the "liveness video capture" screen 
-VIDEO_FACIAL_CAPTURE_STEP_1 - User reached the 1st challenge during "liveness video capture", challenge_type can be found in eventProperties 
-VIDEO_FACIAL_CAPTURE_STEP_2 - User reached the 2nd challenge during "liveness video capture", challenge_type can be found in eventProperties 
-VIDEO_FACIAL_CAPTURE_CONFIRMATION - User reached the "liveness video confirmation" screen 
-VIDEO_FACIAL_UPLOAD - User's liveness video is uploading
+screen_welcome - User reached the "Welcome" screen 
+screen_document_front_capture_file_upload - User reached the "document capture" screen for the front side (for two-sided or one-sided document) 
+screen_document_back_capture_file_upload - User reached the "document capture" screen for the back side (for two-sided document) 
+screen_document_front_confirmation - User reached the "document confirmation" screen for the front side (for two-sided or one-sided document) 
+screen_document_back_confirmation - User reached the "document confirmation" screen for the back side (for two-sided document) 
+Starting_upload - User's file is uploading 
+Completed upload - User's file uploaded successfully
```

Comment:
Not a blocker for this PR but I initially thought they might've been typos...until I realised this is how it is our code. 😅
We should fix this to be consistent with the other event names IMO @stefaniacardenas thoughts?